### ISSUE TITLE
Misuse of URLs. Please use URI instead.

### ISSUE THREAD

**Issue by aef** [keybase/keybase-issues#1481]
When I check a Keybase ID or track it, with the CLI client like this:

```
keybase id github://aef
keybase track twitter://someone
```

The double slash "//" and the general syntax gives the impression that I have to write some kind of URL here.  A URL is a kind of URI that describes a resource by its location and is mainly differentiated from other URIs by the aforementioned double slashes after the scheme part. The only problem is, that in this case, no location is described, but just the name of the resource, in this case a user's name.

I think the intention of having a kind of URI here is obvious and a it is indeed a good idea. But I would rather prefer to actually having a generic URI than having a fake URL. I would suggest using something like the following:

```
keybase id github:aef
keybase track twitter:someone
```

For more information see: https://en.wikipedia.org/wiki/URI_scheme

**Comment by maxtaco** [comment]
Good point.  In our new Go implementation the "//" is optional, but you make a good case that it shouldn't be there at all.  Thanks!
